src/common/components/ChevronDownIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { AUTHENTIC_BLUE_900 } from '../colors';
+
+interface Props {
+    className?: string;
+}
+
+export const ChevronDownIcon: React.FC<Props> = props => {
+    return (
+        <svg color={AUTHENTIC_BLUE_900} width={24} height={24} viewBox="0 0 24 24" {...props}>
+            <path d="M12 13.726l7-6.175v2.676l-7 6.222-7-6.222V7.55z" fill="currentColor" fillRule="nonzero" />
+        </svg>
+    );
+};
```

Comment:
For consistency, we could use the same syntax as in: src/ActionBar/Icons/Export.tsx

Reply:
The icons in the `ActionBar` all have to have a certain interface due to how the `ActionBar` works. This icon is copied from the internal design system, so we could potentially replace this icon in the future.